ci/beautify.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+echo "------- Beautify Script --------"
+RV=0
+for FILE_PATH in `find $TUSK_REPO_PATH -type f -path "$TUSK_REPO_PATH/src/*" -or -type f -path "$TUSK_REPO_PATH/services/*" -or -type f -path "$TUSK_REPO_PATH/tests/*"`
+do
+    RV_PRESERVE=0
+    if [[ $FILE_PATH =~ .*\.(js|html|css) ]]; then
+        js-beautify -f $FILE_PATH -q | diff -q $FILE_PATH - 
+        RV_PRESERVE=$?
+        if [[ $RV_PRESERVE != 0 ]]; then
+            RV=$RV_PRESERVE
+        else
+            echo "$FILE_PATH is Beautiful :)"
+        fi
+    fi
+done
+echo "------- End Beautify Script --------"
+exit $RV
```

Comment:
Is it possible to get this stuff done in JavaScript instead of shell? It's already a javascript project and we're using node and npm, even travis can get it done with javascript - maybe we can even move this logic to `.travis.yml`?

Reply:
[It seems that it is possible to get this stuff done in Javascript](https://www.npmjs.com/package/js-beautify) instead of the shell, but I'm not sure you'd want to do that. By performing this functionality in javascript you:

1. Shoehorn yourself into JS only tools that happen to expose their underlying libraries OR 
2. you have to subprocess the tools anyways, which is functionally the same as the shell in this case.

We can't move it to .travis.yml, `find`/`exec` doesn't play nice with pipes so I cannot reduce it to one-line for a .travis.yml command. I could do it with temporary files, but that just is a redundant step.

Ultimately, I have no stake in this. If you want to force your build environment into javascript, go for it, but be aware of what you are doing.